File: database/port/FeuilleMatch.java
package database.port;

//[using]

@Table(name = "feuille_match")
class FeuilleMatch
{

    @Column(name = "id_feuille_match")
    public int idFeuilleMatch;    

    @Column(name = "id_match")
    public String idMatch;    

    @Column(name = "id_equipe_joueur")
    public String idEquipeJoueur;    



    public void setIdFeuilleMatch(int value){
        this.idFeuilleMatch = value;
    }
    public int getIdFeuilleMatch(){
        return this.idFeuilleMatch;
    }

    public void setIdMatch(String value){
        this.idMatch = value;
    }
    public String getIdMatch(){
        return this.idMatch;
    }

    public void setIdEquipeJoueur(String value){
        this.idEquipeJoueur = value;
    }
    public String getIdEquipeJoueur(){
        return this.idEquipeJoueur;
    }

   
}

File: database/port/FeuilleMatch.cs
namespace database.port;

//[using]

[Table(name = "feuille_match")]
class FeuilleMatch
{

    [Column(name = "id_feuille_match")]
    public Int idFeuilleMatch { get; set; }    

    [Column(name = "id_match")]
    public String idMatch { get; set; }    

    [Column(name = "id_equipe_joueur")]
    public String idEquipeJoueur { get; set; }    



    public void SetIdFeuilleMatch(Int value){
        this.idFeuilleMatch = value;
    }
    public Int GetIdFeuilleMatch(){
        return this.idFeuilleMatch;
    }

    public void SetIdMatch(String value){
        this.idMatch = value;
    }
    public String GetIdMatch(){
        return this.idMatch;
    }

    public void SetIdEquipeJoueur(String value){
        this.idEquipeJoueur = value;
    }
    public String GetIdEquipeJoueur(){
        return this.idEquipeJoueur;
    }

   
}
